core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -308,6 +308,7 @@ object Boot extends IOApp with LazyLogging {
       val submissionCostService: SubmissionCostService =
         SubmissionCostService.constructor(
           gcsConfig.getString("billingExportTableName"),
```

Comment:
These values are added to rawls.conf in https://github.com/broadinstitute/firecloud-develop/pull/3051